lib/pysixdb.py

```diff
@@ -143,3 +143,10 @@ def close(self):
         '''Disconnect the database'''
         self.conn.commit()
         self.conn.close()
+
+    def __del__(self):
+        try:
+            self.conn.commit()
+            self.conn.close()
+        except Exception:
+            pass
```

Comment:
Is there any technical reason to add a destructor here and remove the one in study?.
And also this destructor is duplicated with `close` method.

Reply:
if for whatever reason the Study.db attribute is deleted without the Study object being deleted, Study's __del__ method would not be called and the database connection would not be closed. Having the __del__ method "lower down" is safer for fringe cases.

Reply:
Ok, it does make sense, we can keep it. And please add some messages in this destructor to indicate the status.